ui/src/basics/Json.tsx

```diff
@@ -0,0 +1,30 @@
+import ReactJson, { ReactJsonViewProps } from "react-json-view";
+
+export const Json = ({ src }: ReactJsonViewProps) => (
+  <ReactJson
+    src={src}
+    name={null}
+    collapseStringsAfterLength={15}
+    displayDataTypes={false}
+    displayObjectSize={false}
+    collapsed={false}
+    theme={{
+      base00: "#292d3e",
```

Comment:
Since we're just using 2 different colors and repeating them, let's cache these as vars. Also,  let's use SDS color definitions here (https://github.com/stellar/stellar-design-system/blob/main/%40stellar/design-system/src/global.scss):

for ex:
```
const COLOR_PALETTE = {
background:  var(--pal-example-code),
color: var(--pal-brand-primary-on); // this is the color I'm using in the Error Log, pretty close to what you have here
}
```

Reply:
Can we use the `var(--...)` syntax in JavaScript? I thought it was scss syntax.

Reply:
You're right, I wasn't paying attention - we would need to do a workaround to evaluate the css var as a JS string (which we should do, eventually). 

I would still save those hex values as vars, though, to prevent rewriting the same strings multiple times